test/dhcp/record_test.rb

```diff
@@ -1,6 +1,5 @@
 require 'test_helper'
 require 'dhcp_common/dhcp_common'
-require 'dhcp_common/server'
```

Comment:
This is necessary to run the test file standalone, else it errors with: `NameError: uninitialized constant Proxy::DHCP::Subnet`